{
   "active_lock_reason" : null,
   "assignee" : null,
   "assignees" : [],
   "author_association" : "MEMBER",
   "body" : "https://cirrus-ci.com/task/5524545770094592?logs=ci#L3290\r\n\r\n\r\n\r\n",
   "closed_at" : null,
   "closed_by" : null,
   "comments" : 3,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29806/comments",
   "created_at" : "2024-04-04T14:13:57Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29806/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/29806",
   "id" : 2225634991,
   "labels" : [
      {
         "color" : "08a781",
         "default" : false,
         "description" : null,
         "id" : 149424,
         "name" : "Wallet",
         "node_id" : "MDU6TGFiZWwxNDk0MjQ=",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Wallet"
      },
      {
         "color" : "d4c5f9",
         "default" : false,
         "description" : null,
         "id" : 62963516,
         "name" : "Tests",
         "node_id" : "MDU6TGFiZWw2Mjk2MzUxNg==",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Tests"
      },
      {
         "color" : "cccccc",
         "default" : false,
         "description" : "",
         "id" : 5334691551,
         "name" : "CI failed",
         "node_id" : "LA_kwDOABII588AAAABPfju3w",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/CI%20failed"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29806/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "node_id" : "I_kwDOABII586EqH6v",
   "number" : 29806,
   "performed_via_github_app" : null,
   "reactions" : {
      "+1" : 0,
      "-1" : 0,
      "confused" : 0,
      "eyes" : 0,
      "heart" : 0,
      "hooray" : 0,
      "laugh" : 0,
      "rocket" : 0,
      "total_count" : 0,
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29806/reactions"
   },
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "open",
   "state_reason" : null,
   "timeline_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29806/timeline",
   "title" : "intermittent issue in wallet_backwards_compatibility.py:  line 245, in run_test                                        assert txs[3][\"abandoned\"]                                    AssertionError",
   "updated_at" : "2024-04-27T15:01:30Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29806",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/6399679?v=4",
      "events_url" : "https://api.github.com/users/maflcko/events{/privacy}",
      "followers_url" : "https://api.github.com/users/maflcko/followers",
      "following_url" : "https://api.github.com/users/maflcko/following{/other_user}",
      "gists_url" : "https://api.github.com/users/maflcko/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/maflcko",
      "id" : 6399679,
      "login" : "maflcko",
      "node_id" : "MDQ6VXNlcjYzOTk2Nzk=",
      "organizations_url" : "https://api.github.com/users/maflcko/orgs",
      "received_events_url" : "https://api.github.com/users/maflcko/received_events",
      "repos_url" : "https://api.github.com/users/maflcko/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/maflcko/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/maflcko/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/maflcko"
   }
}
